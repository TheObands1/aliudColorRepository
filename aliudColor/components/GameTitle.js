import React from "react";
import { Text, View, StyleSheet } from "react-native";

const GameTitle = () => {
    return(
        <View style={{ flexDirection: "row" }}>
            <Text style={[TitleStyle.fontStyle, { color: "#ecf0f1" }]}>aliud</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#E64C3C" }, { textTransform: 'uppercase'}]}>c</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#E57E31" }]}>o</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#F1C431" }]}>l</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#68CC73" }]}>o</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#3998DB" }]}>r</Text>
        </View>
    );
}

const TitleStyle = StyleSheet.create({
    fontStyle: {
      fontSize: 50,
      fontFamily: "dogbyte"
    }
   });

export { GameTitle };