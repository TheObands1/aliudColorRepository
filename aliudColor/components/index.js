export * from './GameTitle'
export * from "./GameplayComponents/GameGrid"
export * from "./GameplayComponents/LoseView"
export * from "./GameplayComponents/pauseComponents/pauseView"
export * from "./GameplayComponents/bottomComponents/BottomArea"