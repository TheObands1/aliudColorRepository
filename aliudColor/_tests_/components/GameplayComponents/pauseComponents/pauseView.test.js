import React from 'react';
import renderer from 'react-test-renderer';
import { PauseView } from "../../../../components"

describe('tests of the pauseView', () => {
    it('Create the pause view and expect it to have 2 children, which represent the paused image and text', () => {
      const tree = renderer.create(<PauseView />).toJSON();
      expect(tree.children.length).toBe(2);
    });

    it('Render the component correctly', () => {
        const tree = renderer.create(<PauseView />).toJSON();
        expect(tree).toMatchSnapshot();
      });

  });