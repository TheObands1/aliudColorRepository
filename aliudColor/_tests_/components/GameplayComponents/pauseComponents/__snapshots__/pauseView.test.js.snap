// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests of the pauseView Render the component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 80,
        "width": 80,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#eee",
        "fontFamily": "dogbyte",
        "fontSize": 60,
        "marginTop": 20,
        "textAlign": "center",
      }
    }
  >
    paused
  </Text>
</View>
`;
